apps/coscrad-frontend/src/components/footer/footer.spec.tsx

```diff
@@ -40,3 +40,48 @@ describe('Footer', () => {
         expect(screenRes).toBeTruthy();
     });
 });
+
+describe('when an email is provided', () => {
+    const dummyConfigurableContent = getDummyConfigurableContent();
+
+    it('should render the email icon and address', () => {
+        const email = 'johndoe@johndoe.com';
+        const myConfigurableContent = { ...dummyConfigurableContent, email };
+
+        renderWithProviders(
+            <MemoryRouter>
+                <Footer />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const searchPattern = new RegExp(email);
+        const screenRes = screen.getByText(searchPattern);
+
+        expect(screenRes).toBeTruthy();
```

Comment:
Keep what is here and also add a check that the ICON is rendered (just the logical opposite of what you have below).